extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/FastCamelContext.java

```diff
@@ -81,7 +81,10 @@ public String getVersion() {
     @Override
     protected Registry createRegistry() {
         // Registry creation is done at build time
-        throw new UnsupportedOperationException();
+        throw new UnsupportedOperationException(
```

Comment:
I add a text into the `UnsupportedOperationException`, which is thrown in case that context is re-started with `CamelQuarkusTestSupport`. This will make the exception understandable for users. @ppalaga What do you think?